app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -491,90 +490,170 @@ class CampaignStacExport(
             .TaskGeoJSON(geojson)
         )
       } getOrElse Map.empty
       // make the label item
-      taskExtent <- TaskDao
-        .createUnionedGeomExtent(
-          annotationProject.id,
-          inputState.exportDefinition.taskStatuses
-        )
-        .transact(xa)
-      labelItemExtensionO <- AnnotationProjectDao
-        .getAnnotationProjectStacInfo(annotationProject.id)
-        .transact(xa)
-      labelItemsAppend = (taskExtent, labelItemExtensionO, sceneItemO) mapN {
-        case (extent, labelItemExtension, sceneItem) =>
+      taskExtent <-
+        TaskDao
+          .createUnionedGeomExtent(
+            annotationProject.id,
+            inputState.exportDefinition.taskStatuses
+          )
+          .transact(xa)
+      labelItemExtensionO <-
+        AnnotationProjectDao
+          .getAnnotationProjectStacInfo(annotationProject.id)
+          .transact(xa)
+      labelItemsAppend = (taskExtent, labelItemExtensionO, imageryItemO) mapN {
+        case (extent, labelItemExtension, imageryItem) =>
           Map(
             newtypes.AnnotationProjectId(annotationProject.id) -> makeLabelItem(
               extent,
-              sceneCreationTime map { _.toInstant },
+              root.datetime
+                .as[Instant]
+                .getOption(imageryItem.value.properties.asJson) orElse Some(
+                Instant.now
+              ),
               labelItemExtension,
-              sceneItem
+              imageryItem
             )
           )
       } getOrElse Map.empty
       // make the label collection for this annotation project with an s3 asset
     } yield {
       (appendLabelItem(labelItemsAppend) `compose` popAnnotationProject(
         newtypes.AnnotationProjectId(annotationProject.id)
-      ) `compose` appendSceneItem(
-        sceneItemsAppend
+      ) `compose` appendImageryItem(
+        imageryItemsAppend
       ) `compose` appendLabelAssets(labelAssetAppend))(
         inputState
       )
     }
 
   }
 
+  private def makeTileLayersItem(
+      tileLayers: List[TileLayer],
+      extent: Extent,
+      createdAt: Instant
+  ): newtypes.SceneItem = {
+    val assets = Map(tileLayers map { layer =>
+      layer.name -> StacItemAsset(
+        layer.url,
+        Some(layer.name),
+        Some(s"${layer.layerType} tiles"),
+        Set(StacAssetRole.Data),
+        layer.layerType match {
+          case MVT =>
+            Some(VendorMediaType("application/vnd.mapbox-vector-tile"))
+          case TMS => Some(`image/png`)
+        }
+      )
+    }: _*)
+    newtypes.SceneItem(
+      StacItem(
+        s"${UUID.randomUUID}",
+        stacVersion,
+        Nil,
+        "Feature",
+        extent.toPolygon,
+        TwoDimBbox(
+          extent.xmin,
+          extent.ymin,
+          extent.xmax,
+          extent.ymax
+        ),
+        Nil,
+        assets,
+        None,
+        Map("datetime" -> createdAt.asJson).asJsonObject
+      )
+    )
+  }
```

Comment:
This is where we actually make the item. Two things to point out --

1. There are no links. We don't need them. The links are already handled in `writeImageryItems`.
2. There's no collection id. We add that in `writeImageryitems` also. No special handling is required, since `imageryCollectionId` is available at that point, so we can fill it in when it's important to do so.
3. No special handling in `writeImageryItems` is required for making sure we don't sign the tms urls, since we only sign the url for an asset called `data`, and even then only if it's in the bucket.